server/src/main/java/com/linecorp/centraldogma/server/storage/StorageManager.java

```diff
@@ -96,23 +96,16 @@ default T create(String name, Author author) {
     T unremove(String name);
 
     /**
-     * Returns a set of names for the elements which have been purged.
+     * Purges a set of names for the elements which have been marked for purging.
      */
-    Set<String> listPurged();
+    void purgeMarked();
 
     /**
-     * Purges an element with the specified {@code name}.
+     * Marks for purging an element with the specified {@code name}.
```

Comment:
Marks an element with the specified name for purge.

Reply:
`purge` looks better. Thanks. :-)